examples/pytorch/README.md

```diff
@@ -4,10 +4,21 @@ This directory shows a simple example of how to use Mountpoint for Amazon S3 as
 It trains a ResNet-50 model for a few epochs using [PyTorch Lightning](https://www.pytorchlightning.ai/index.html), with synthetic ImageNet-style training data, stored in S3 as shards in WebDataset format.
 The idea is that because Mountpoint exposes your S3 bucket as a local file system, you can just use standard file-oriented loaders rather than writing S3-specific code, and get great performance.
 
+## Getting started
+
 To get started, launch an EC2 instance with a GPU (we used a [g5.2xlarge](https://aws.amazon.com/ec2/instance-types/g5/)),
 choosing the [AWS Deep Learning AMI GPU PyTorch 2.0.1 (Amazon Linux 2)](https://aws.amazon.com/releasenotes/aws-deep-learning-ami-gpu-pytorch-2-0-amazon-linux-2/) as your AMI.
```

Comment:
Should we change the recommendation for the AMI to [Deep Learning OSS Nvidia Driver AMI GPU PyTorch 2.3 (Amazon Linux 2)](https://aws.amazon.com/releasenotes/aws-deep-learning-ami-gpu-pytorch-2-3-amazon-linux-2/)? It is the current default on AWS console.

Reply:
I'd prefer to revalidate the results before making such a change (and I'm having a bit of an issue getting the setup working). I'd suggest we can make a follow-up change for this if its OK.